samples/templates/machines.yaml

```diff
@@ -17,13 +17,14 @@ spec:
     apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
     namespace: default
     name: test-cluster-kube-master-01
-  version: "v1.15.0"
+  version: "1.15.0"
```

Comment:
Could you update kubernetesVersion in L56 as well?

Reply:
@hidekazuna done